tests/functional/tnf/test_tnf_node_replacement.py

```
@@ -0,0 +1,347 @@
+
+        while time.time() - start_wait < max_wait:
+            try:
+                pending_csrs = OCP(kind="csr").get().get("items", [])
+                for csr in pending_csrs:
+                    csr_name = csr["metadata"]["name"]
+                    status = csr.get("status", {})
+                    if not status.get("conditions"):
+                        ocp_node.exec_oc_cmd(f"adm certificate approve {csr_name}")
+                        logger.info(f"Approved CSR {csr_name}")
+            except Exception:
+                pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)